modules/migration/base-migration.js

```diff
@@ -0,0 +1,61 @@
+const appRootPath = require('app-root-path');
+const path = require('path');
+const fs = require('fs-extra');
+
+const MIGRATION_FOLDER_NAME = 'migrations';
+
+class BaseMigration {
+    constructor(migrationName, logger, config) {
+        if (!migrationName || migrationName === '') {
```

Comment:
Please consider removing || migrationName === '' since it will be covered within !migrationName check